ScoutSuite/providers/gcp/resources/iam/service_accounts.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from ScoutSuite.providers.base.configs.resources import CompositeResources
+from ScoutSuite.providers.gcp.resources.iam.bindings import Bindings
+from ScoutSuite.providers.gcp.resources.iam.keys import Keys
+
+class ServiceAccounts(CompositeResources):
+    _children = [
+        ('bindings', Bindings),
+        ('keys', Keys)
+    ]
+
+    def __init__(self, gcp_facade, project_id):
+        self.gcp_facade = gcp_facade
+        self.project_id = project_id
+
+    async def fetch_all(self):
+        raw_service_accounts = await self.gcp_facade.iam.get_service_accounts(self.project_id)
+        for raw_service_account in raw_service_accounts:
+            service_account_id, service_account = self._parse_service_account(raw_service_account)
+            self[service_account_id] = service_account
+        await self._fetch_children()
+    
+    async def _fetch_children(self):
+        for service_account_id, service_account in self.items():
```

Comment:
You'll need to parallelize this loop (see https://github.com/nccgroup/ScoutSuite/pull/229/files#r263374926) and probably make a refactoring of `fetch_children` as well (would be probably worth it to define a `GCPCompositeResources` to do so, like `AWSCompositeResources` and `AzureCompositeResources` ([here](https://github.com/nccgroup/ScoutSuite/blob/28e6d1f3e4bd4b068c0b0caf31eb718743919d3c/ScoutSuite/providers/aws/resources/resources.py#L25) and [there](https://github.com/nccgroup/ScoutSuite/blob/28e6d1f3e4bd4b068c0b0caf31eb718743919d3c/ScoutSuite/providers/azure/resources/resources.py#L8))).

Reply:
Done!